apps/src/lib/kits/maker/ui/EligibilityChecklist.story.jsx

```diff
@@ -72,7 +73,7 @@ export default storybook => {
         )
       },
       {
```

Comment:
Let's add stories to this file for the scenario where the academic year PD requirement has not been passed.

Reply:
Done!